inspire_classifier/app.py

```diff
@@ -43,8 +44,10 @@ def force_type(cls, rv, environ=None):
 def create_app():
     Flask.response_class = JsonResponse
     app = Flask(__name__)
+    app_prefix = 'CLASSIFIER'
     app.config['CLASSIFIER_BASE_PATH'] = app.instance_path
     app.config.from_object('inspire_classifier.config')
+    app.config.update({key: val for key, val in os.environ.items() if key.startswith(app_prefix)})
```

Comment:
this won't work if you need a var not to be a string though.

Reply:
and do I want a var which is not a string? especially that all env variables there are prefixed

Reply:
plus we do same thing in all our apps which are using `invenio_config_loader` 
https://github.com/inveniosoftware/invenio-config/blob/a23b96f0cee86b100646f7df05880086881663b3/invenio_config/env.py#L33-L35

Reply:
They later eval it which is not such a great idea imho https://github.com/inveniosoftware/invenio-config/blob/a23b96f0cee86b100646f7df05880086881663b3/invenio_config/env.py#L43

Reply:
So what do you suggest? Updating config with every variable? For me it doesn't make sense since we don't have any variable which is not string (and we're not planning to do it) and it's not a very common practice to set env variables that are not string.